{
  "cve": "CVE-2021-28378",
  "mitre": {
    "cpes": [],
    "created": "2021-03-15T05:20:50+00:00",
    "description": "Gitea 1.12.x and 1.13.x before 1.13.4 allows XSS via certain issue data in some situations.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 3.7,
        "vector": "CVSS:3.1/AC:H/AV:N/A:N/C:L/I:L/PR:L/S:U/UI:R"
      },
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2021/28xxx/CVE-2021-28378.json",
    "references": [
      "https://blog.gitea.io/2021/03/gitea-1.13.4-is-released/",
      "https://github.com/PandatiX/CVE-2021-28378",
      "https://github.com/go-gitea/gitea/pull/14898"
    ],
    "title": null,
    "updated": "2024-08-03T21:40:14.222000+00:00",
    "vendors": [],
    "weaknesses": []
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:a:gitea:gitea:*:*:*:*:*:*:*:*"
    ],
    "created": "2021-03-15T06:15:12.423000+00:00",
    "description": "Gitea 1.12.x and 1.13.x before 1.13.4 allows XSS via certain issue data in some situations.",
    "metrics": {
      "cvssV2_0": {
        "score": 3.5,
        "vector": "AV:N/AC:M/Au:S/C:N/I:P/A:N"
      },
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 5.4,
        "vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:C/C:L/I:L/A:N"
      },
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2021/CVE-2021-28378.json",
    "references": [
      "https://blog.gitea.io/2021/03/gitea-1.13.4-is-released/",
      "https://github.com/PandatiX/CVE-2021-28378",
      "https://github.com/go-gitea/gitea/pull/14898"
    ],
    "title": null,
    "updated": "2021-12-16T18:20:22.583000+00:00",
    "vendors": [
      "gitea",
      "gitea$PRODUCT$gitea"
    ],
    "weaknesses": [
      "CWE-79"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [
        "cpe:2.3:a:gitea:gitea:*:*:*:*:*:*:*:*"
      ],
      "providers": [
        "nvd"
      ]
    },
    "created": {
      "data": "2021-03-15T05:20:50+00:00",
      "provider": "mitre"
    },
    "description": {
      "data": "Gitea 1.12.x and 1.13.x before 1.13.4 allows XSS via certain issue data in some situations.",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {
          "score": 3.5,
          "vector": "AV:N/AC:M/Au:S/C:N/I:P/A:N"
        },
        "provider": "nvd"
      },
      "cvssV3_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_1": {
        "data": {
          "score": 3.7,
          "vector": "CVSS:3.1/AC:H/AV:N/A:N/C:L/I:L/PR:L/S:U/UI:R"
        },
        "provider": "mitre"
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": null,
        "provider": null
      }
    },
    "references": {
      "data": [
        "https://blog.gitea.io/2021/03/gitea-1.13.4-is-released/",
        "https://github.com/PandatiX/CVE-2021-28378",
        "https://github.com/go-gitea/gitea/pull/14898"
      ],
      "providers": [
        "mitre",
        "nvd"
      ]
    },
    "title": {
      "data": null,
      "provider": null
    },
    "updated": {
      "data": "2024-08-03T21:40:14.222000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [
        "gitea",
        "gitea$PRODUCT$gitea"
      ],
      "providers": [
        "nvd"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-79"
      ],
      "providers": [
        "nvd"
      ]
    }
  }
}